packages/fxa-payments-server/src/routes/Product/SubscriptionCreate/index.scss

```diff
@@ -4,29 +4,49 @@
 .product-payment {
   background: $color-white;
   border: 1px solid #e7e7e7;
+  border-bottom: 0;
+  flex: 1;
   margin: 30px 0;
```

Comment:
I think this `margin` + no height is causing some problems in tablet:
<img width="844" alt="image" src="https://user-images.githubusercontent.com/13018240/77358464-e44abc80-6d17-11ea-8992-46375c362fe6.png">

You might want to create a `src/styles/_variables.scss` for payments, import it into a `src/styles/_main.scss`, and import `@import './styles/main';` into `src/index.scss`. If you create:
 
```
$header-height: 64px;
$header-height-tablet: 81px;
```

... Then we can firstly take off the weird padding on `body` to get these values looking not as messy since we have spacing everywhere:
```
body {
  padding: 0;
}
```

```
.main-content {
  padding-top: $header-height;
  ...
  @include min-width('tablet') {
    padding-top: $header-height-tablet;
    ...
}

```
Then you can do this:
```
.product-payment: {
  ...
  // take `margin` out we don't need it (I didn't check mobile though, maybe you need `margin-top` there)
  min-height: calc(100vh - $header-height);

  @include min-width('tablet') {
    min-height: calc(100vh - $header-height-tablet);
    ...
```

This way, our values are more exact, and the spacing at the top stays the same even if we change around the header height. 
![image](https://user-images.githubusercontent.com/13018240/77360006-93889300-6d1a-11ea-91aa-2af8b8ded2a1.png)